src/components/PersonItem/PersonItem.tsx

```diff
@@ -0,0 +1,43 @@
+import { useParams } from 'react-router-dom';
+import { Person } from '../../types';
+import { PersonLink } from '../PersonLink';
+import classNames from 'classnames';
+import { useContext } from 'react';
+import { PeopleContext } from '../../peopleContext';
+import { getPersonByName } from '../../utils/getPeopleByName';
+
+type Props = {
+  person: Person;
+};
+
+export const PersonItem: React.FC<Props> = ({ person }) => {
+  const { slug: userId, sex, born, died, fatherName, motherName } = person;
+  const { slug: selectedUserId } = useParams();
+
+  const people = useContext(PeopleContext);
+  const mother = getPersonByName(motherName, people);
+  const father = getPersonByName(fatherName, people);
```

Comment:
You're using the `getPersonByName` function to find the mother and father of the person. This could potentially be a performance issue if there are a lot of people in the context. A better approach would be to store the people in a map data structure with the name as the key. This would make finding a person by name a constant time operation.